hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -136,6 +136,7 @@ export const UnconnectedCheckedOutTable = ({
                                     >
```

Comment:
Right above this line (github won't let me comment on it), `styles.widthButton` is undefined